backend/items/location_test.py

```
@@ -0,0 +1,527 @@
+        LocationService.validate_depth(parent, max_depth=10)
+        
+        # Should fail at level 10
+        too_deep = Location(user=self.alice, display_name="Level 10", parent=parent)
+        with self.assertRaises(ValidationError):
+            LocationService.validate_depth(too_deep, max_depth=10)
+
+    def test_get_location_tree(self):
+        """Verify location tree structure is built correctly"""
+        home = Location.objects.create(user=self.alice, slug="home", display_name="Home")
+        bedroom = Location.objects.create(user=self.alice, slug="bedroom", display_name="Bedroom", parent=home)
+        closet = Location.objects.create(user=self.alice, slug="closet", display_name="Closet", parent=bedroom)
```

Ruff (0.15.2)

[error] Local variable `closet` is assigned to but never used

Remove assignment to unused variable `closet`

(F841)